scripts/npm-publish.sh

```
@@ -12,5 +12,6 @@ if npm view "${PKG_NAME}@${PKG_VER}" version > /dev/null 2>&1; then
   echo "already published: ${PKG_NAME}@${PKG_VER} — skipping"
 else
   bun pm pack
-  npm publish ./$PKG_NAME-$PKG_VER.tgz --provenance --access public $flags
+  cd $project_dir
+  npm publish $project_dir/lib/distopia/$PKG_NAME-$PKG_VER.tgz --verbose --provenance --access public $flags
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)